routes.yaml

```diff
@@ -0,0 +1,7 @@
+app_name: customer-feedback-api
+group: api
+
+weight: 900
+routes:
+  1: ^/customer-feedback
+  2: ^/customer-feedback/healthcheck
```

Comment:
Is this needed if the service is going directly to ECS?

Reply:
The healthcheck or the routes.yml?

Reply:
Removed